packages/api/src/index.ts

```
@@ -1,15 +1,27 @@
 import express, { Express, Request, Response, NextFunction } from 'express';
-import { HealthStatus, ApiResponse, API_ENDPOINTS, PathRequest, RouteResponse, RouteCalculationResponse, RoutePathNode, RouteSegment } from '@campus-nav/shared/types';
+import { HealthStatus, ApiResponse, API_ENDPOINTS, RouteResponse, RouteCalculationResponse, RoutePathNode, RouteSegment } from '@campus-nav/shared/types';
 import { PathfindingService } from './pathfinding';
-import { config, isDevelopment } from './config';
-import { createDatabaseService } from './db/connection';
+import { config, isDevelopment, isProduction } from './config';
+import { createDatabaseService, DatabaseConfig } from './db/connection';
 import createHealthRouter from './routes/health';
+import { validate } from './middleware/validate';
+import { pathfindSchema } from './schemas/pathfind.schema';
```

ESLint

[error] 'pathfindSchema' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)